handbook/engineering/hiring/engineering-manager-web.md

```diff
@@ -1,5 +1,7 @@
 # Engineering Manager - Web
 
+**[Apply here.](https://jobs.lever.co/sourcegraph/13286143-746e-4381-8049-94cf9911c2ab)**
```

Comment:
Do we really want this at the top? I want candidates to read the JD before applying.

Reply:
Okay without it being at the top.